api/testing.py

```diff
@@ -81,6 +82,9 @@ def initialize_adobe(self, vendor_id_library, short_token_libraries=[]):
             ConfigurationSetting.for_library_and_externalintegration(
                 self._db, ExternalIntegration.PASSWORD, library, self.registry
             ).value = secret
+            ConfigurationSetting.for_library_and_externalintegration(
+                self._db, RegistrationConstants.LIBRARY_REGISTRATION_STATUS, library, self.registry
+            ).value = RegistrationConstants.SUCCESS_STATUS
```

Comment:
Existing tests were expecting a valid registration, so here we set the needed status.